org.sonarlint.eclipse.ui/src/org/sonarlint/eclipse/ui/internal/views/locations/IssueLocationsView.java

```diff
@@ -114,7 +114,11 @@ public boolean equals(Object obj) {
 
     public FlowRootNode(MarkerFlow flow) {
       this.flow = flow;
-      children = flow.getLocations().stream().map(FlowNode::new).collect(toList());
+      children = flow.getLocations().stream()
+        // SLE-388 - "Highlight-only" locations don't have a message
+        .filter(l -> l.getMessage() != null && !"".equals(l.getMessage()))
```

Comment:
Minor comment, we already have a StringUtils class that does that:

```suggestion
        .filter(l -> !StringUtils.isEmpty(l.getMessage()))
```